client/client_test.go

```diff
@@ -319,10 +323,10 @@ func TestClientGetAndPut(t *testing.T) {
 	db := createTestClient(s.ServingAddr())
 
 	value := []byte("value")
-	if err := db.Put("key", value); err != nil {
+	if err := db.Put(testUser+"/key", value); err != nil {
 		t.Fatalf("unable to put value: %s", err)
 	}
-	gr, err := db.Get("key")
+	gr, err := db.Get(testUser + "/key")
```

Comment:
lots of tests for permissions enforcement here and there, just not specifically for test-user. I'll probably clean some of those up as I go through the rest of the tests adding it.